<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswordDialog</class>
 <widget class="QDialog" name="ChangePasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Input old password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="oldPassword"/>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Input new password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="newPassword"/>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Repeat new password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="repeatPassword"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="IncorrectOldPassword">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>37</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="IncorrectNewPassword">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>37</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>209</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(227, 227, 227);
font: 16pt &quot;Oswald&quot;;
color: rgb(0, 0, 0);
border-radius: 10;</string>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
